app/src/main/java/com/sun/premierleague/ui/team/TeamPresenter.kt

```diff
@@ -0,0 +1,62 @@
+package com.sun.premierleague.ui.team
+
+import com.sun.premierleague.R
+import com.sun.premierleague.data.model.Coache
+import com.sun.premierleague.data.model.Player
+import com.sun.premierleague.data.model.TeamItem
+import com.sun.premierleague.data.source.FootballRepository
+import com.sun.premierleague.data.source.remote.OnDataLoadedCallback
+import java.lang.Exception
+
+class TeamPresenter(
+    private val view: TeamContact.View,
+    private val repository: FootballRepository
+) : TeamContact.Presenter {
+
+    override fun getTeamInformation(idTeam: String) {
+        repository.getTeam(idTeam, object : OnDataLoadedCallback<List<TeamItem>> {
+            override fun onSuccess(data: List<TeamItem>) {
+                if (data.isEmpty()) {
+                    view.showError(R.string.err_no_data)
+                    return
+                }
+                view.showTeam(data)
+
+                val coaches: List<Coache> = data.first().coaches
```

Comment:
Conventions có suggest viết rút gọn như này nhé 
```suggestion
                val coaches = data.first().coaches
```